gradle/libs.versions.toml

```diff
@@ -16,21 +16,20 @@
 [versions]
 accompanist = "0.28.0"
 androidx-datastore = "1.0.0"
-androidx-navigation = "2.5.3"
-androidx-window = "1.1.0-alpha03"
+androidx-navigation = "2.6.0"
+androidx-window = "1.1.0-alpha02"
```

Comment:
Why is the downgrade?

Reply:
Oh. I first upgrade it to 1.1.0 but too many breaking changes (hence #36) and I revert it to the wrong version. I will fix it